policy/README.md

```diff
@@ -0,0 +1,353 @@
+
+There are potential complexities that are as-yet unaddressed, such as the latency between a Provider deciding to enter such an area, and the number of vehicles in the area when the scooters arrive.  As this specification is Alpha, feedback and refinements are welcome.
+
+The payload returned from a GET call to the `value_url` will have the following immutable fields:
+
+| Name         | Type      | R/O | Description |
+| ----         | ----      | --- | ----------- |
+| `value`      | integer   | R   | Value of whatever the rule measures |
+| `timestamp`  | timestamp | R   | Timestamp the value was recorded |
+| `policy_id`  | UUID      | R   | Relevant `policy_id` for reference |
+
+MDS will likely include some sort of streaming mechanism in the upcoming releases that may supplant or replace this mechanism.
+
+<a name="endpoints"></a>
+## Endpoints
+
+The provider-facing Policy API consists of the following two endpoints.
+
+`GET /policies`
+
+Parameters:
+
+| Name         | Type      | R/O | Description |
+| ----         | ----      | --- | ----------- |
+| `start_time` | timestamp | O   | Earliest effective date, default=effective now |
+| `end_time`   | timestamp | O   | Latest effective date, default=all future |
+
+Note: provider_id is an implicit parameter and will be encoded in the authentication key
+
+Returns: List of policy objects effective during the timespan described by greater than or equal to `start_time` and less than or equal to `end_time`
+
+Policies will be returned in order of effective date (see Policy Schema, below), with pagination as in Agency or Provider.
+
+`GET /policies/{id}`
+
+Returns: One policy object whose id is `id`
+
+Parameters: none
+
+`GET /geographies/{id}`
+
+Returns: One immutable GeoJSON FeatureCollection with an embedded UUID identifier property of `id`
+
+Parameters: none
+
+Note: Intentionally omitted `GET /geographies` until a compelling use-case can be identified.
+ 
+<a name="authoring"></a>
+## Authoring
+
+Creating, editing, and publishing policies may be performed via a variety of mechanisms, and are therefore not specified here.  Authoring tools would optionally provide schema extensions for tooling, including author, Provider-specificity, etc.  We may add a specific instance of such extensions in a later revision of this document, but at present that’s TBD.
+
+One mechanism may be via a source-control repository, where pull-requests to Policy objects are proposed, left open to public commentary, etc., and served as static content via the endpoints listed above.
```

Comment:
I'd remove this couple lines, as this is more relevent for a PR message rather than for actual inclusion in the spec.

Reply:
Line 196 is meant to address your static-served-files ask.  Were you asking to lose line 194 too?